rustls/src/tls13/key_schedule.rs

```diff
@@ -770,6 +770,21 @@ impl KeySchedule {
     }
 }
 
+/// [HKDF-Expand-Label] where the output is an AEAD key.
```

Comment:
Reword commit message? (perhaps "move `derive_traffic_key` up in preparation for becoming public")